### DISCUSSION TITLE
How can i set my localhost virtual host name for google authentication

### DISCUSSION THREAD

**Discussion by mdrakash** [laravel/framework#49409] [General]
i want to set callback url to my local server URL for google authentication

**Comment by chuckrincon** [comment]
Try using [ngrok](https://ngrok.com/) or [expose](https://expose.dev/) to achieve this.
Both tools have user-friendly commands to open your localhost to the internet.

Here are some examples:
[Setting Up Laravel with Ngrok: A Step-by-Step Guide](https://arjunamrutiya.medium.com/setting-up-laravel-with-ngrok-a-step-by-step-guide-a565b2c430b5)

[Sharing sites via Expose](https://herd.laravel.com/docs/1/advanced-usage/sharing-sites)

**Reply by mdrakash** [reply]
is possible to use local url to redirect
